upload-chunked-file.js

```diff
@@ -0,0 +1,43 @@
+function chunkFileUpload(file, serverUrl, chunkSize) {
```

Comment:
It's a good idea to validate input parameters, such as ensuring that `chunkSize` is a positive number and `file` is a valid File object. This will help prevent runtime errors.